src/Http/Middleware/ShareInertiaData.php

```diff
@@ -20,6 +20,10 @@ class ShareInertiaData
      */
     public function handle($request, $next)
     {
+        Inertia::version(function () {
+            return md5_file(public_path('mix-manifest.json'));
```

Comment:
What if this file didn't exist? Wouldn't that make Laravel automatically 500?

Reply:
Yup.

Reply:
Good point. I'll do some more testing on this, including seeing if I can find a good solution for Vapor.

Reply:
For Vapor, you can use `config('app.asset_url')`, but my point is that this shouldn't be on vendor code.

Reply:
Yeah, I already had a fix for it ready, I just didn't get around to testing it yet as I got interrupted by some other things. I just pushed the changes. Do you still think it's a bad idea now?

I could move it to a publishable Service Provider, but then it'd always boot. 
The nice thing about the current solution (middleware) is that it's only booted when it's actually being used.